numba/cuda/kernels/transpose.py

```diff
@@ -48,6 +52,6 @@ def kernel(input, output):
     blocks = int(b.shape[0]/tile_height + 1), int(b.shape[1]/tile_width + 1)
     # one thread per tile element
     threads = tile_height, tile_width
-    kernel[blocks, threads](a, b)
+    kernel[blocks, threads, a.stream](a, b)
```

Comment:
This change is causing an error in `numba.cuda.tests.cudapy.test_smart_array.TestJIT`. The array `a` can be either a `DeviceNDArray`or a numpy array, which does not have the `.stream` attribute.